README.md

```diff
@@ -8,27 +8,23 @@
 ```sh
 $ npm install -g standard-changelog
 $ cd my-project
-$ standard-changelog CHANGELOG.md -w
+$ standard-changelog
 ```
 
-This will *not* overwrite any previous changelog. The above generates a changelog based on commits since the last semver tag that match the pattern of a "Feature", "Fix", "Performance Improvement" or "Breaking Changes".
+The above generates a changelog based on commits since the last semver tag that match the pattern of a "Feature", "Fix", "Performance Improvement" or "Breaking Changes".
 
-If you first time use this tool and want to generate all previous changelog, you could do
+If you first time use this tool and want to generate all previous changelog, you can run:
 
 ```sh
-$ standard-changelog -i CHANGELOG.md -w -r 0
+$ standard-changelog -r 0
```

Comment:
I think `--releaseCount 0` is somewhat confusing. I like the boolean flag approach of `--first-release` used in `conventional-recommended-workflow` better. Perhaps we could alias `--first-release` or `-f` to mean the same thing as `-r 0`?